lib/models/ObjectMD.js

```diff
@@ -1028,6 +1030,33 @@ class ObjectMD {
         return this;
     }
 
+    /**
+     * Create or update the microVersionId field
+     *
+     * This field can be used to force an update in MongoDB (in case
+     * no other metadata field changes) or to manage conflicts during
+     * concurrent updates, using conditions on the microVersionId
+     * field.
+     *
+     * It's a base64-encoded field of 64 bits randomly generated
+     *
+     * @return {ObjectMD} itself
+     */
+    updateMicroVersionId() {
+        this._data.microVersionId = crypto.randomBytes(8).toString('hex');
+        return this;
+    }
+
+    /**
+     * Get the microVersionId field, or null if not set
+     *
+     * @return {string|null} the microVersionId field if exists, or
+     * {null} if it does not exist
+     */
+    getMicroVersionId() {
```

Comment:
Way outside the scope of this change but javascript now has get/set methods for classes https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set that would do wonders here.

Reply:
Going to resolve this now since there isnt really any action to take here, but will keep it in mind if any refactoring time is allocated.

Reply:
@rahulreddy fyi